builtin/logical/pkiext/pkiext_binary/pki_cluster.go

```diff
@@ -109,6 +109,11 @@ func (vpc *VaultPkiCluster) GetActiveNode() *api.Client {
 	return vpc.GetActiveClusterNode().APIClient()
 }
 
+// GetCACertPEM returns the Vault cluster's PEM-encoded CA certificate.
+func (vpc *VaultPkiCluster) GetCACertPEM() []byte {
```

Comment:
Maybe `GetListenerCACertPEM()` to disambiguate it?

Reply:
I like that suggestion. I was trying to come up with a distinguishing name, but was drawing blanks.